backend/server.py

```diff
@@ -45,77 +45,142 @@
 data = [
 {
     "id": 1,
-    "jobCategory": "Project Management",
+    "jobCategory": "IT, Software & Digital",
     "jobCategoryDescription": "Delivering exciting, innovative, complex, and technical projects.",
-    "availableJobs": [
+    "subCategories": [
         {
-            "jobId": "1a",
-            "jobTitle": "Scrum Master",
-            "jobDescription": "Scrum Master is responsible for ensuring that the team follows the rules of Scrum, and helps the team to continuously improve their process.",
-            "jobSkills": ["Scrum", "Agile", "Kanban", "Jira"]
+            "subCategoryId": "1a",
+            "subCategoryName": "IT Business Analyst",
+            "subCategoryDescription": "Focused on leveraging business analytics tools and methodologies to improve decision-making and business practices within IT, Software & Digital projects.",
+            "availableJobs": [
+                {
+                    "jobId": "WD-0012443",
```

Comment:
For now, maybe the job is can be like 1a_1 or anything, to relevant to the if used in this data structure